### PR TITLE
fix: use github token for getTags API to increase the rate limit

### PR DESCRIPTION
**Pull request by JivusAyrus**
#### Checklist

- [ ] run `make test`
- [x] PR title must follow [conventional-commit-standard](https://github.com/wundergraph/wundergraph/blob/main/CONTRIBUTING.md#conventional-commit-standard)
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [x] commit message and code follows the [Code of conduct](https://github.com/wundergraph/wundergraph/blob/main/CODE_OF_CONDUCT.md)

### CODE REVIEW

**Review by fiam — APPROVED** [review]
LGTM

File: packages/create-wundergraph-app/src/helpers/getRepoTags.ts [hunk 1]
```diff
@@ -1,12 +1,19 @@
 import got from 'got';
 
 import { getRepoInfo } from './getRepoInfo';
+import dotenv from 'dotenv';
+
+dotenv.config();
```
Comment (StarpTech) [inline packages/create-wundergraph-app/src/helpers/getRepoTags.ts:6]: We should document that we support .env

**Review by StarpTech — CHANGES_REQUESTED** [review]

File: packages/create-wundergraph-app/src/helpers/getRepoTags.ts (hunk 1 above)
Comment (JivusAyrus) [reply packages/create-wundergraph-app/src/helpers/getRepoTags.ts:6]: Added an example.env file.
Should I mention it in the README?

Comment (StarpTech) [reply packages/create-wundergraph-app/src/helpers/getRepoTags.ts:6]: Yes, a simple sentence.

Comment (JivusAyrus) [reply packages/create-wundergraph-app/src/helpers/getRepoTags.ts:6]: Done 👍

**Review by jensneuse — APPROVED** [review]

**Review by StarpTech — APPROVED** [review]
LGTM

**Pull request merged**